camera_matrix:
  rows: 3
  cols: 3
  data:
  - 589.6079022397957
  - 0.0
  - 366.4966179976604
  - 0.0
  - 590.179073412096
  - 297.9873639512045
  - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
  rows: 1
  cols: 5
  data:
  - -0.24896938465800103
  - 0.1343538476339651
  - 0.00032043791813315855
  - -0.0003614084621200428
  - -0.06579839078949333
projection_matrix:
  rows: 4
  cols: 4
  data:
  - -0.6806163187161818
  - 0.7324682188871416
  - 0.01586615950793227
  - -0.016788995687316167
  - 0.5744517108313589
  - 0.546976537166916
  - -0.6089514756627279
  - -0.02269247299737612
  - -0.45471601975316167
  - -0.4053479691686974
  - -0.7930487786199881
  - 0.5454465438837742
  - 0.0
  - 0.0
  - 0.0
  - 1.0
